src/infer/browsers/inferFirefoxVersion.ts

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import * as log from 'loglevel';
+
+type FirefoxVersions = {
+  FIREFOX_AURORA: string;
+  FIREFOX_DEVEDITION: string;
+  FIREFOX_ESR: string;
+  FIREFOX_ESR_NEXT: string;
+  FIREFOX_NIGHTLY: string;
+  LAST_MERGE_DATE: string;
+  LAST_RELEASE_DATE: string;
+  LAST_SOFTFREEZE_DATE: string;
+  LATEST_FIREFOX_DEVEL_VERSION: string;
+  LATEST_FIREFOX_OLDER_VERSION: string;
+  LATEST_FIREFOX_RELEASED_DEVEL_VERSION: string;
+  LATEST_FIREFOX_VERSION: string;
+  NEXT_MERGE_DATE: string;
+  NEXT_RELEASE_DATE: string;
+  NEXT_SOFTFREEZE_DATE: string;
+};
+
+const FIREFOX_VERSIONS_URL =
+  'https://product-details.mozilla.org/1.0/firefox_versions.json';
+
+export async function getLatestFirefoxVersion(
+  url = FIREFOX_VERSIONS_URL,
+): Promise<string> {
+  log.debug('Grabbing apple version data from', url);
+  const response = await axios.get(url, { timeout: 5000 });
+  if (response.status !== 200) {
+    throw new Error(`Bad request: Status code ${response.status}`);
+  }
```

Comment:
Error handling: can you wrap this in a try/catch and return a default version that is fetched from a new `DEFAULT_FIREFOX_VERSION` defined in `const.ts` (visible next to `DEFAULT_ELECTRON_VERSION` , with a comment reminding to periodically bump this to whatever's the latest Firefox) ?